pyproject.toml

```diff
@@ -1,4 +1,30 @@
+[project]
+name = "esp-eth-drivers"
+description = "ESP32 family Ethernet drivers collection"
+readme = "README.md"
+version="1.0.0"
```

Comment:
Is `version` required field? `esp-eth-drivers` is monorepo. Hence each component has its own version.